<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>387</width>
          <height>433</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>0</y>
           <width>381</width>
           <height>101</height>
          </rect>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>70</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 19pt &quot;Algerian&quot;;</string>
          </property>
          <property name="text">
           <string>Welcome back!</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;.VnArial&quot;;
</string>
         </property>
         <property name="text">
          <string>Email:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>321</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
         </property>
         <property name="placeholderText">
          <string>Enter your email</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;.VnArial&quot;;</string>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>321</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
         </property>
         <property name="placeholderText">
          <string>Enter your password</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>250</y>
           <width>111</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;.VnArial&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Keep me log in</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>280</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>Log in</string>
         </property>
        </widget>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>320</y>
           <width>241</width>
           <height>37</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;.VnArial&quot;;
</string>
            </property>
            <property name="text">
             <string>Need an account?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="styleSheet">
             <string notr="true">border-radius: 0px;
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 85, 255);
background-color: rgb(0, 0, 0);
</string>
            </property>
            <property name="text">
             <string>Register</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="pushButton_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>360</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 0px;
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 85, 255);
background-color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string>Forgot password?</string>
         </property>
        </widget>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>90</y>
           <width>300</width>
           <height>200</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>300</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>    Play</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../.designer/image/icon2.png</normaloff>../../../.designer/image/icon2.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>300</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);</string>
         </property>
         <property name="text">
          <string>    Stop</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../.designer/image/icon3.png</normaloff>../../../.designer/image/icon3.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>300</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(0, 85, 255);</string>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../.designer/image/image-removebg-preview (7).png</normaloff>../../../.designer/image/image-removebg-preview (7).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>161</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Screenshot 2024-03-03 at 08.21.47.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
